playbooks/gcp/openshift-cluster/build_image.yml

```diff
@@ -0,0 +1,109 @@
+---
```

Comment:
Should we rename aws/gcp files to match?  I think the aws equivalent is `build_ami.yml`.

Also, should we move most of the heavy lifting under the `roles/openshift_gce/build_image.yml` so that the logic is under the role?

Reply:
AMI is Amazon specific, GCE always calls it an image.  I would argue for `build_image` for both before `build_ami` for both.

Reply:
I think maybe yes, but this was going to be a future refactor point.

Reply:
Do you mind if I refactor this as a follow up?

Reply:
That sounds fine.